### PR TITLE
Complete profile changes

### PR DESCRIPTION
**Pull request by sijuman**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by erich-wang** [comment]
@sijuman, what the purpose of resource types declared in API version? Do you expect to generate code for these resource types, no more and no less? If this is the case, I have some questions:
1.	Some resource types do not exist in spec but appear in profile, e.g. "locations/accountOperationResults" for Microsoft.Batch; 
2.	Some resource types are defined in spec but do not appear in profile, e.g. "/subscriptions/{subscriptionId}/providers/Microsoft.CognitiveServices/skus" for Microsoft.CognitiveServices, autorest doesn't support to exclude some resource types from one input spec file per my understanding
3. Some services contain more than one API versions in the profile, e.g. compute, what if there are duplicate resource types among these API versions?

**Comment by sijuman** [comment]
@erich-wang If there are any duplicates with different api-versions, always pick the latest api version for that resource type.

**Comment by erich-wang** [comment]
> @erich-wang If there are any duplicates with different api-versions, always pick the latest api version for that resource type.

Currently autorest doesn't support this scenario. I think one possible solution is: you provide sdk team the following info for each service:
service name, supported api version, json file list of each supported api version.

**Comment by sijuman** [comment]
There shouldn't be any duplicates in the profile map file, and the file has the supported api version for each resource provider and the specified types.

**Comment by erich-wang** [comment]
As I said, currently autorest only supports .json as minimal input instead of resource types. There's no way to exclude some of resource types defined in .json in generated code. We prefer to have format like https://github.com/Azure/azure-rest-api-specs/blob/master/profile/2019-03-01-hybrid.json

**Comment by sijuman** [comment]
@sphibbs @bganapa There shouldn't be any resources that are left out. The difference between the two profiles are the path is not included and this profile includes which clouds are supported which is needed.

**Comment by erich-wang** [comment]
@sijuman, @sphibbs @bganapa  I'll have to regenerate workable json file which is similar to [2019-03-10-hybrid.json](https://github.com/Azure/azure-rest-api-specs/blob/master/profile/2019-03-01-hybrid.json) based on 2019-07-01-profile.json. We have to have agreement on that workable json file before generating SDKs.
@sijuman , another question is: are you going to add tag in readme*.md? Normally it is service team's responsibility to update these readme*.md.

**Comment by sijuman** [comment]
Please let me know when to review the workale json file.

Yes I can update the tag in the readme file if needed. Thanks for your help!

**Review by mmyyrroonn — CHANGES_REQUESTED** [review]
- line 258, I can just find 2018-07-12-preview for BotService.
- line 865, it's a strange format. I cannot find these version "2015-05-01" and "2017-10-01"
- line 1332, for network, "2018-05-01" doesn't exist 
- line 1517,  "2016-08-10", "2018-01-10", "2017-07-01" don't exist 

@MyronFanQiu "line 258" for Microsoft.botservice, while it is in preview folder, the version is listed as 2018-07-12 inside of the file without the "-preview" appended to it. I don't believe this should be changed

"line 865" https://github.com/Azure/azure-rest-api-specs/tree/master/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2017-10-01

"line 1332" https://github.com/Azure/azure-rest-api-specs/tree/master/specification/dns/resource-manager/Microsoft.Network/stable/2018-05-01

"line 1517" https://github.com/Azure/azure-rest-api-specs/tree/master/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable

This contains all three api versions. While they may go under different folders, they belong to the same resource provider.

**Comment by MyronFanQiu** [comment]
@sijuman Hello. It's will be much more helpful for SDK owners to have path information in the profile.json.

File: profile/2019-07-01-profile.json [hunk 1]
```diff
@@ -163,18 +133,6 @@
...
-            "AzureChina"
-          ]
-        },
-        "2017-05-01": {
-          "operations": [
-            "AzureGov",
-            "AzureChina"
-          ]
-        },
         "2018-05-01": {
```
[the remark this answers is not in the document]
Comment (sijuman) [reply profile/2019-07-01-profile.json]: @MyronFanQiu https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2018-05-01

[the remark this answers is not in the document]
Comment (MyronFanQiu) [reply profile/2019-07-01-profile.json]: @sijuman Please check the line 865. I think the format is not correct.

[the remark this answers is not in the document]
Comment (sijuman) [reply profile/2019-07-01-profile.json]: @MyronFanQiu this should be fixed now

**Comment by erich-wang** [comment]
@sijuman , the following profiles do not exist, please fix

Microsoft.AnalysisServices:2017-08-01-beta
Microsoft.Authorization:2015-06-01
Microsoft.Authorization:2018-05-01
Microsoft.BotService:2018-07-12
Microsoft.Cache:2018-03-01
Microsoft.CostManagement:2018-05-31
Microsoft.DBforMySQL:2017-12-01-preview
Microsoft.DBforPostgreSQL:2017-12-01-preview
Microsoft.Devices:2018-12-01-preview
Microsoft.Features:2015-12-01
Microsoft.ImportExport:2016-11-01
microsoft.insights:2015-05-01
microsoft.insights:2017-10-01
Microsoft.Media:2018-07-01
Microsoft.Media:2015-10-01
Microsoft.Network:2018-05-01
Microsoft.PowerBI:2016-01-29
Microsoft.RecoveryServices:2016-08-10
Microsoft.RecoveryServices:2018-01-10
Microsoft.RecoveryServices:2017-07-01
Microsoft.Solutions:2018-06-01

**Comment by sijuman** [comment]
@erich-wang Below is specified in the order asked. 
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/analysisservices/resource-manager/Microsoft.AnalysisServices/preview/2017-08-01-beta
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/authorization/resource-manager/Microsoft.Authorization/preview/2015-06-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2018-05-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/redis/resource-manager/Microsoft.Cache/stable/2018-03-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2018-05-31
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2017-12-01-preview
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2018-12-01-preview
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2016-11-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2017-10-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/mediaservices/resource-manager/Microsoft.Media/stable/2015-10-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/dns/resource-manager/Microsoft.Network/stable/2018-05-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/powerbiembedded/resource-manager/Microsoft.PowerBI/stable
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2016-08-10
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-01-10
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2017-07-01
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager/Microsoft.Solutions/stable/2018-06-01

**Comment by MyronFanQiu** [comment]
@sijuman Looks like line 865 still has the formatting error.

**Comment by sijuman** [comment]
Pushed the latest changes for the resource type. Please let me know if there are any further changes needed. @erich-wang

**Comment by erich-wang** [comment]
@sijuman, I could not find the following res types:

Microsoft.Compute:2017-03-30:virtualMachineScaleSets/networkInterfaces
Microsoft.Compute:2017-03-30:virtualMachineScaleSets/virtualMachines/networkInterfaces
Microsoft.Compute:2017-03-30:virtualMachineScaleSets/publicIPAddresses
Microsoft.NotificationHubs:2017-04-01:checkNameAvailability
Microsoft.Sql:2017-10-01-preview:servers/databases/VulnerabilityAssessment/scans

**Comment by sijuman** [comment]
@erich-wang Pushed the final changes

**Comment by erich-wang** [comment]
@sijuman, this file [2019-07-01-profile_with_spec_file.zip](https://github.com/Azure/azure-rest-api-specs/files/3577297/2019-07-01-profile_with_spec_file.zip) contains corresponding spec file for each resource types, next step is to update readme.*.md to define 2019-07-01 profile for auto generation for languages you want.

**Comment by erich-wang** [comment]
@sijuman , come out the following file lists for each service after cross-check. I just created pull request #7141 as sample for authorization service, please note I do not touch readme.*.md for ruby/go/python, you need to contact these sdk owners to check what configuration change is needed.

```
adhybridhealthservice:
Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

advisor:
Microsoft.Advisor/stable/2017-04-19/advisor.json

alertsmanagement:
Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json

analysisservices:
Microsoft.AnalysisServices/preview/2017-08-01-beta/analysisservices.json

apimanagement:
Microsoft.ApiManagement/stable/2019-01-01/apimapis.json
Microsoft.ApiManagement/stable/2019-01-01/apimdeployment.json

authorization:
Microsoft.Authorization/preview/2015-06-01/authorization-ClassicAdminCalls.json
Microsoft.Authorization/preview/2018-01-01-preview/authorization-RoleDefinitionsCalls.json
Microsoft.Authorization/preview/2018-07-01-preview/authorization-DenyAssignmentGetCalls.json

resources:
Microsoft.Authorization/stable/2018-05-01/policyAssignments.json
Microsoft.Authorization/stable/2018-05-01/policyDefinitions.json
Microsoft.Authorization/stable/2018-05-01/policySetDefinitions.json
Microsoft.Features/stable/2015-12-01/features.json
Microsoft.Resources/stable/2016-06-01/subscriptions.json
Microsoft.Resources/stable/2016-09-01/links.json
Microsoft.Resources/stable/2016-09-01/resources.json
Microsoft.Solutions/stable/2018-06-01/managedapplications.json

automation:
Microsoft.Automation/stable/2015-10-31/account.json

azurestack:
Microsoft.AzureStack/stable/2017-06-01/AzureStack.json
Microsoft.AzureStack/stable/2017-06-01/CustomerSubscription.json
Microsoft.AzureStack/stable/2017-06-01/Product.json

batch:
Microsoft.Batch/stable/2019-04-01/BatchManagement.json

batchai:
Microsoft.BatchAI/stable/2018-03-01/BatchAI.json
Microsoft.BatchAI/stable/2018-05-01/BatchAI.json

botservice:
Microsoft.BotService/preview/2018-07-12/botservice.json

redis:
Microsoft.Cache/stable/2018-03-01/redis.json

cognitiveservices:
Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json

compute:
Microsoft.Compute/stable/2018-09-30/disk.json
Microsoft.Compute/stable/2018-10-01/compute.json
Microsoft.Compute/stable/2018-10-01/runCommands.json
Microsoft.Compute/stable/2019-03-01/gallery.json

consumption:
Microsoft.Consumption/preview/2018-11-01-preview/consumption.json
Microsoft.Consumption/stable/2018-06-30/consumption.json

containerregistry:
Microsoft.ContainerRegistry/preview/2016-06-27-preview/containerregistry.json
Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json

cost-management:
Microsoft.CostManagement/stable/2018-05-31/costmanagement.json

databox:
Microsoft.DataBox/stable/2018-01-01/databox.json

datafactory:
Microsoft.DataFactory/stable/2018-06-01/datafactory.json

datamigration:
Microsoft.DataMigration/preview/2018-07-15-preview/datamigration.json

mysql:
Microsoft.DBforMySQL/preview/2017-12-01-preview/mysql.json

postgresql:
Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json

iothub:
Microsoft.Devices/preview/2018-12-01-preview/iothub.json

eventhub:
Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json
Microsoft.EventHub/stable/2014-09-01/EventHub.json
Microsoft.EventHub/stable/2017-04-01/EventHub.json

storageimportexport:
Microsoft.ImportExport/stable/2016-11-01/storageimportexport.json

monitor:
microsoft.insights/preview/2017-05-01-preview/diagnosticsSettingsCategories_API.json
microsoft.insights/stable/2015-04-01/autoscale_API.json
microsoft.insights/stable/2015-04-01/eventCategories_API.json
microsoft.insights/stable/2015-04-01/operations_API.json
microsoft.insights/stable/2016-03-01/alertRulesIncidents_API.json
microsoft.insights/stable/2016-03-01/logProfiles_API.json
microsoft.insights/stable/2017-04-01/activityLogAlerts_API.json
microsoft.insights/stable/2018-01-01/metricDefinitions_API.json
microsoft.insights/stable/2018-01-01/metrics_API.json
microsoft.insights/stable/2018-03-01/metricAlert_API.json
microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json
microsoft.insights/stable/2018-09-01/actionGroups_API.json
microsoft.insights/stable/2018-09-01/baseline_API.json
microsoft.insights/stable/2018-09-01/calculateBaseline_API.json

applicationinsights:
microsoft.insights/preview/2017-10-01/componentFeaturesAndPricing_API.json
microsoft.insights/preview/2017-10-01/eaSubscriptionMigration_API.json
microsoft.insights/preview/2018-06-17-preview/workbooks_API.json
microsoft.insights/stable/2015-05-01/webTests_API.json

keyvault:
Microsoft.KeyVault/preview/2018-02-14-preview/keyvault.json
Microsoft.KeyVault/preview/2018-02-14-preview/providers.json
Microsoft.KeyVault/preview/2018-02-14-preview/secrets.json

logic:
Microsoft.Logic/preview/2018-07-01-preview/logic.json

marketplaceordering:
Microsoft.MarketplaceOrdering/stable/2015-06-01/Agreements.json

mediaservices:
Microsoft.Media/stable/2015-10-01/media.json
Microsoft.Media/stable/2018-07-01/AccountFilters.json
Microsoft.Media/stable/2018-07-01/Accounts.json
Microsoft.Media/stable/2018-07-01/AssetsAndAssetFilters.json
Microsoft.Media/stable/2018-07-01/ContentKeyPolicies.json
Microsoft.Media/stable/2018-07-01/Encoding.json
Microsoft.Media/stable/2018-07-01/StreamingPoliciesAndStreamingLocators.json
Microsoft.Media/stable/2018-07-01/streamingservice.json

migrate:
Microsoft.Migrate/stable/2018-02-02/migrate.json

network:
Microsoft.Network/stable/2019-02-01/applicationGateway.json
Microsoft.Network/stable/2019-02-01/applicationSecurityGroup.json
Microsoft.Network/stable/2019-02-01/availableDelegations.json
Microsoft.Network/stable/2019-02-01/azureFirewall.json
Microsoft.Network/stable/2019-02-01/azureFirewallFqdnTag.json
Microsoft.Network/stable/2019-02-01/checkDnsAvailability.json
Microsoft.Network/stable/2019-02-01/ddosCustomPolicy.json
Microsoft.Network/stable/2019-02-01/ddosProtectionPlan.json
Microsoft.Network/stable/2019-02-01/endpointService.json
Microsoft.Network/stable/2019-02-01/expressRouteCircuit.json
Microsoft.Network/stable/2019-02-01/expressRouteCrossConnection.json
Microsoft.Network/stable/2019-02-01/loadBalancer.json
Microsoft.Network/stable/2019-02-01/networkProfile.json
Microsoft.Network/stable/2019-02-01/networkSecurityGroup.json
Microsoft.Network/stable/2019-02-01/networkWatcher.json
Microsoft.Network/stable/2019-02-01/operation.json
Microsoft.Network/stable/2019-02-01/publicIpAddress.json
Microsoft.Network/stable/2019-02-01/publicIpPrefix.json
Microsoft.Network/stable/2019-02-01/routeFilter.json
Microsoft.Network/stable/2019-02-01/serviceCommunity.json
Microsoft.Network/stable/2019-02-01/serviceEndpointPolicy.json
Microsoft.Network/stable/2019-02-01/usage.json
Microsoft.Network/stable/2019-02-01/virtualNetwork.json
Microsoft.Network/stable/2019-02-01/virtualNetworkGateway.json
Microsoft.Network/stable/2019-02-01/virtualNetworkTap.json
Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json

dns:
Microsoft.Network/stable/2018-05-01/dns.json

trafficmanager:
Microsoft.Network/stable/2018-04-01/trafficmanager.json

notificationhubs:
Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json

operationalinsights:
Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json

operationsmanagement:
Microsoft.OperationsManagement/preview/2015-11-01-preview/OperationsManagement.json

policyinsights:
Microsoft.PolicyInsights/preview/2018-07-01-preview/policyStates.json
Microsoft.PolicyInsights/preview/2018-07-01-preview/policyTrackedResources.json
Microsoft.PolicyInsights/preview/2018-07-01-preview/remediations.json
Microsoft.PolicyInsights/stable/2018-04-04/policyEvents.json

portal:
Microsoft.Portal/preview/2015-08-01-preview/portal.json

powerbiembedded:
Microsoft.PowerBI/stable/2016-01-29/powerbiembedded.json

powerbidedicated:
Microsoft.PowerBIDedicated/stable/2017-10-01/powerbidedicated.json

recoveryservicessiterecovery:
Microsoft.RecoveryServices/stable/2016-08-10/service.json

recoveryservices:
Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

recoveryservicesbackup:
Microsoft.RecoveryServices/stable/2017-07-01/bms.json

relay:
Microsoft.Relay/stable/2017-04-01/relay.json

resourcehealth:
Microsoft.ResourceHealth/stable/2015-01-01/resourcehealth.json
Microsoft.ResourceHealth/stable/2017-07-01/resourcehealth.json

scheduler:
Microsoft.Scheduler/stable/2016-03-01/scheduler.json

search:
Microsoft.Search/stable/2015-08-19/search.json

servicebus:
Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json
Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

servicefabric:
Microsoft.ServiceFabric/stable/2018-02-01/cluster.json

sql:
Microsoft.Sql/preview/2015-05-01-preview/advisors.json
Microsoft.Sql/preview/2015-05-01-preview/databaseAutomaticTuning.json
Microsoft.Sql/preview/2015-05-01-preview/encryptionProtectors.json
Microsoft.Sql/preview/2015-05-01-preview/failoverGroups.json
Microsoft.Sql/preview/2015-05-01-preview/serverKeys.json
Microsoft.Sql/preview/2015-05-01-preview/syncAgents.json
Microsoft.Sql/preview/2015-05-01-preview/syncGroups.json
Microsoft.Sql/preview/2015-05-01-preview/syncMembers.json
Microsoft.Sql/preview/2015-05-01-preview/virtualclusters.json
Microsoft.Sql/preview/2015-05-01-preview/virtualNetworkRules.json
Microsoft.Sql/preview/2017-03-01-preview/blobAuditing.json
Microsoft.Sql/preview/2017-03-01-preview/cancelOperations.json
Microsoft.Sql/preview/2017-03-01-preview/databaseVulnerabilityAssessmentBaselines.json
Microsoft.Sql/preview/2017-03-01-preview/jobs.json
Microsoft.Sql/preview/2017-03-01-preview/longTermRetention.json
Microsoft.Sql/preview/2017-03-01-preview/ManagedBackupShortTermRetention.json
Microsoft.Sql/preview/2017-03-01-preview/managedDatabases.json
Microsoft.Sql/preview/2017-03-01-preview/managedInstanceAdministrators.json
Microsoft.Sql/preview/2017-03-01-preview/serverAutomaticTuning.json
Microsoft.Sql/preview/2017-03-01-preview/serverDnsAliases.json
Microsoft.Sql/preview/2017-03-01-preview/serverSecurityAlertPolicies.json
Microsoft.Sql/preview/2017-10-01-preview/cancelPoolOperations.json
Microsoft.Sql/preview/2017-10-01-preview/capabilities.json
Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json
Microsoft.Sql/preview/2017-10-01-preview/instanceFailoverGroups.json
Microsoft.Sql/preview/2017-10-01-preview/managedDatabaseVulnerabilityAssesmentRuleBaselines.json
Microsoft.Sql/preview/2017-10-01-preview/ManagedInstanceTdeCertificates.json
Microsoft.Sql/preview/2017-10-01-preview/recoverableManagedDatabases.json
Microsoft.Sql/preview/2017-10-01-preview/TdeCertificates.json
Microsoft.Sql/preview/2018-06-01-preview/DatabaseSchema.json
Microsoft.Sql/preview/2018-06-01-preview/DatabaseSecurityAlertPolicies.json
Microsoft.Sql/preview/2018-06-01-preview/ManagedDatabaseSchema.json
Microsoft.Sql/preview/2018-06-01-preview/ManagedInstanceVulnerabilityAssessments.json
Microsoft.Sql/preview/2018-06-01-preview/ServerVulnerabilityAssessments.json
Microsoft.Sql/stable/2014-04-01/backupLongTermRetentionPolicies.json
Microsoft.Sql/stable/2014-04-01/deprecated.json
Microsoft.Sql/stable/2014-04-01/geoBackupPolicies.json

storage:
Microsoft.Storage/stable/2017-10-01/storage.json
Microsoft.Storage/stable/2019-04-01/blob.json
Microsoft.Storage/stable/2019-04-01/file.json
Microsoft.Storage/stable/2019-04-01/storage.json

storagesync:
Microsoft.StorageSync/stable/2019-02-01/storagesync.json

web:
Microsoft.Web/stable/2015-08-01/service.json
Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json
Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json
Microsoft.Web/stable/2018-02-01/Provider.json
Microsoft.Web/stable/2018-02-01/ResourceProvider.json
Microsoft.Web/stable/2018-02-01/WebApps.json


```

**Review by erich-wang — APPROVED** [review]

**Comment by erich-wang** [comment]
@MyronFanQiu, please approve this PR if you don't have other concern, thanks.

**Review by mmyyrroonn — APPROVED** [review]

**Review by bganapa — APPROVED** [review]

**Pull request merged**